pkg/alertmanager/statefulset.go

```diff
@@ -48,7 +47,7 @@ func makeStatefulSet(am *v1alpha1.Alertmanager, old *v1beta1.StatefulSet, config
 	// Potentially an update handler on first insertion.
 
 	if am.Spec.BaseImage == "" {
-		am.Spec.BaseImage = defaultBaseImage
+		am.Spec.BaseImage = config.AlertmanagerDefaultBaseImage
```

Comment:
When instantiating the Alertmanager operator/controller, the Prometheus config is passed, but an Alertmanager config is instantiated from that config. The Alertmanager config needs to be adapted.